### ISSUE TITLE
Support for ADD directive

### ISSUE THREAD

**Issue by mwkaufman** [noop-cloud/noop-local#17]
```
mkaufman@Michaels-MBP solr % cat Noopfile
COMPONENT solr service
FROM ubuntu:latest
ADD entrypoint.sh /opt
CMD ["/bin/bash", "/opt/entrypoint.sh"]

```
```
mkaufman@Michaels-MBP solr % noop run              
noop-local v1.0.1
Application root found at /Users/mkaufman/Source/manage_wine_noop
Unknown directive 'ADD' at /Users/mkaufman/Source/manage_wine_noop/solr/Noopfile:3

```

https://docs.docker.com/engine/reference/builder/#add

**Comment by mwkaufman** [comment]
In this case, a workaround is to use COPY but I don't believe ADD and COPY are interchangeable in all cases.
https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile

**Comment by arbitrarytech** [comment]
My mistake, I'll get a fix out. Thanks @mwkaufman

**Comment by arbitrarytech** [comment]
Fix is published in `noop-local@1.0.7`
